packages/lesswrong/components/users/DialogueMatchingPage.tsx

```diff
@@ -495,6 +496,10 @@ const useScrollGradient = (ref: React.RefObject<HTMLDivElement>) => {
   return { isScrolledToTop, isScrolledToBottom };
 };
 
+const isHideInRecommendations = (userDialogueChecks: DialogueCheckInfo[], targetUserId: string): boolean => {
+  return userDialogueChecks?.find(check => check.targetUserId === targetUserId)?.hideInRecommendations ?? false;
```

Comment:
The first null-safe access doesn't seem necessary given `userDialogueChecks: DialogueCheckInfo[]` should never be null/undefined.  (The null-safe access after the `find(...)` is, since `find` returns undefined if it doesn't find something.)